plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/engine/openai/OpenAIBaseProperties.java

```diff
@@ -25,6 +25,8 @@ public interface OpenAIBaseProperties extends AIEngineProperties {
 
     String getModel();
 
+    Integer getContextWindowSize();
```

Comment:
I see it's nullable, but it's not annotated with `@Nullable`. Or maybe it's never null, but you assume its return value might be null.